pkg/driverbuilder/docker.go

```diff
@@ -396,20 +397,43 @@ func forwardLogs(logPipe io.Reader) {
 func multiplexedForwardLogs(logPipe io.Reader) {
 	hdr := make([]byte, 8)
 	for {
+		// Load size of message
 		_, err := logPipe.Read(hdr)
 		if err == io.EOF {
 			break
 		}
 		if err != nil {
 			slog.With("err", err.Error()).Error("log pipe error")
+			return
 		}
 		count := binary.BigEndian.Uint32(hdr[4:])
+
+		// Read message
 		dat := make([]byte, count)
-		_, err = logPipe.Read(dat)
-		if err != nil {
-			slog.With("err", err.Error()).Error("log pipe error")
+		var readCnt int
+		for uint32(readCnt) < count {
```

Comment:
Read the message in multiple `Read` until we read all bytes.
For big messages (ie >4096) , they might get split and we need to manage it.